mkdocs.yml

```diff
@@ -44,10 +44,10 @@ nav:
   - Integrations:
       - GitHub: integration/github.md
       - Slack: integration/slack.md
-      - Docker Registries:
-          - Docker Hub: integration/docker-hub.md
-          - Google Container Registry: integration/gcr.md
-      - Google Cloud Platform: integration/gcp.md
+  #    - Docker Registries:
+  #        - Docker Hub: integration/docker-hub.md
```

Comment:
I initially had these commented out but added the placeholder pages because the main page, i.e., the Welcome page, has a link to the Docker Hub integration page. If we comment this out, we need to do something about that link.